### DISCUSSION TITLE
Working with NVivo output

### DISCUSSION THREAD

**Discussion by nickdgardner** [ohtap/ohtap#31] [General]
@ebf77 @jadelintott @Yibing-Du @kmcdono2 
On our OHTAP github, see ohtap/NVivo_output/read_to_df.py for how to read NVivo coding (exported from NVivo as an "extract" .xls file) into a pandas dataframe

**Comment by Yibing-Du** [comment]
@ebf77 @kmcdono2 @nickdgardner @jadelintott 
Here's my weekly update! 
@kmcdono2 I hope you're feeling better! If possible, could you help me figure out how to upload my code to GitHub? I think I don't have permission at the moment.

I cleaned the event extents so that there won't be multiple versions (from different coders) of the same paragraph. Whenever two event extents have overlap (I set the length threshold to 30 characters, though it could be adjusted), we'll keep the longer one. I also calculated # of occurrences & total length. This is compiled as a spreadsheet file: 
[event_extents.xlsx](https://github.com/ohtap/ohtap/files/5863732/event_extents.xlsx)
Here's the basic statistic of coverage by collection: 
<img width="470" alt="Screen Shot 2021-01-24 at 6 18 06 PM" src="https://user-images.githubusercontent.com/27621580/105653458-b753c280-5e70-11eb-81a0-8b38188ae65d.png">

I'll look into by-interviewee trends next week. 
Other than that, I've also started revisiting the encoding task and looking at how to use NVivo output beyond event extents. 

Questions: 
- Is there an alternative format that'll more useful? 
- Should it be organized in order and by groups (e.g. in initiating question - story pairs when applicable)?
- Should we consider coverage in word count or percentage? 
Next steps: 
- Figure out how to clean the texts; currently there're weird symbols and irregular coverage (e.g. the mouse moved a bit more than intended). 
- I tried to find a word count pattern that'll distinguish stories from simple mentions, as Nick proposed; but unfortunately, its distribution doesn't seem to be very useful. 
![length_distribution](https://user-images.githubusercontent.com/27621580/105652658-aace6a80-5e6e-11eb-8600-2904f817bb12.png)
- I'll keep looking for ways to automatically classify the two types, and please comment below with your ideas!

**Comment by ebf77** [comment]
Thanks, Yibing – be sure to enter hours and short sum on the grid:
https://docs.google.com/document/d/1VKdGAcvd4urHEc-iQcEiXubGgyNvwOYVZ9Ej_Tzr9do/edit

Nick, can you trouble-shoot the the uploading to GitHub permission problem?

Good questions below – I’d like to consult with Katie on how to proceed on them.

Best,
Estelle
